Makefile.fem-dbexporter

```diff
@@ -1,44 +1,52 @@
 SHELL := /bin/bash
 
-target:
-	@mkdir -p target
+prepare-fem-dbexporter-mongodb:
+	@mkdir -p target/dbexporter
+	@cp -rn dbexporter/docker-compose-fem-dbexporter-mongodb.yml target/dbexporter
 
-docker-fem-dbexporter: target
-	@cp -n docker-compose-fem-dbexporter.yml target
+prepare-fem-dbexporter-postgresql:
+	@mkdir -p target/dbexporter
+	@cp -rn dbexporter/docker-compose-fem-dbexporter-postgresql.yml target/dbexporter
 
-mongodb: docker-fem-dbexporter
+config-mongodb: prepare-fem-dbexporter-mongodb
 	@echo "mongodb username: "; \
 	read USERNAME; \
-	sed -i "s/{mongodb_username}/$$USERNAME/" target/docker-compose-fem-dbexporter.yml
+	sed -i "s/{mongodb_username}/$$USERNAME/" target/dbexporter/docker-compose-fem-dbexporter-mongodb.yml
 
 	@echo "mongodb password: "; \
 	read -s PASSWORD; \
-	sed -i "s/{mongodb_password}/$$PASSWORD/" target/docker-compose-fem-dbexporter.yml
+	sed -i "s/{mongodb_password}/$$PASSWORD/" target/dbexporter/docker-compose-fem-dbexporter-mongodb.yml
 
 	@echo "mongodb host list (format: host:port[,host:port]): "; \
 	read HOSTS; \
-	sed -i "s/{mongodb_hosts}/$$HOSTS/" target/docker-compose-fem-dbexporter.yml
+	sed -i "s/{mongodb_hosts}/$$HOSTS/" target/dbexporter/docker-compose-fem-dbexporter-mongodb.yml
 
-postgresql: docker-fem-dbexporter
+config-postgresql: prepare-fem-dbexporter-postgresql
 	@echo "postgresql username: "; \
 	read USERNAME; \
-	sed -i "s/{postgresql_username}/$$USERNAME/" target/docker-compose-fem-dbexporter.yml
+	sed -i "s/{postgresql_username}/$$USERNAME/" target/dbexporter/docker-compose-fem-dbexporter-postgresql.yml
 
 	@echo "postgresql password: "; \
 	read -s PASSWORD; \
-	sed -i "s/{postgresql_password}/$$PASSWORD/" target/docker-compose-fem-dbexporter.yml
+	sed -i "s/{postgresql_password}/$$PASSWORD/" target/dbexporter/docker-compose-fem-dbexporter-postgresql.yml
 
 	@echo "postgresql host list (format: host:port[,host:port]): "; \
 	read HOSTS; \
-	sed -i "s/{postgresql_hosts}/$$HOSTS/" target/docker-compose-fem-dbexporter.yml
+	sed -i "s/{postgresql_hosts}/$$HOSTS/" target/dbexporter/docker-compose-fem-dbexporter-postgresql.yml
 
-config: mongodb postgresql
+start-mongodb: prepare-fem-dbexporter-mongodb
+	@echo "Please ensure to call 'config-mongodb' before starting"
+	docker-compose --file target/dbexporter/docker-compose-fem-dbexporter-mongodb.yml up -d
 
-start: docker-fem-dbexporter
-	docker-compose --file target/docker-compose-fem-dbexporter.yml up -d
+stop-mongodb: prepare-fem-dbexporter-mongodb
+	docker-compose --file target/dbexporter/docker-compose-fem-dbexporter-mongodb.yml down
 
-stop: docker-fem-dbexporter
-	docker-compose --file target/docker-compose-fem-dbexporter.yml down
+start-postgresql: prepare-fem-dbexporter-postgresql
+	@echo "Please ensure to call 'config-postgresql' before starting"
+	docker-compose --file target/dbexporter/docker-compose-fem-dbexporter-postgresql.yml up -d
```

Comment:
@ronnyfriedland , I think the definition of the project name is needed now, e.g. add: "--project-name fem".
With the separation into modules it must be ensure that all components run in the same network.
As you know without configure the project name, the name for the container as well as the name for the network were created from the directory name by default. That results into different subnets for the containers. The avoid that we have to set the project name and remove the "fem_"-prefix in all docker-compose files.